SEP-0001.md

```diff
@@ -44,16 +58,19 @@ New SEPs currently undergoing discussion are pull requests into the sunpy-SEP re
 # SEP Template
 Also stored in SEP-template.md
 
-# Header
-* **author(s)**: First Last, First Last
-* **contact email**: me@myemail.org
-* **date-creation**: YYYY-MM-DD
-* **date-last-revision**: YYYY-MM-DD
-* **type**: standard/process
-* **status**: accepted/rejected
-* **discussion**: link to discussion on mailing list
+# SEP-num -- SEP Title
+
+| SEP           | num |
+|---------------|---|
+| title         | SEP Title |
+| author(s)     | First Last, First Last |
```

Comment:
Ah, of course, that makes sense.  Maybe instead write as `First Last[, First Last,...]`